src/Rules/Methods/CallMethodsRule.php

```diff
@@ -31,12 +34,33 @@ public function getNodeType(): string
 
 	public function processNode(Node $node, Scope $scope): array
 	{
-		if (!$node->name instanceof Node\Identifier) {
-			return [];
+		if ($node->name instanceof Node\Identifier) {
+			$methodNames = [$node->name->name];
+		} else {
```

Comment:
I see there are a lot of rules which early exit on :

```
if (!$node->name instanceof Node\Identifier) {
			return [];
}
```

maybe these are easy to enhance with a similar check I did here.
(just supporting the `getConstantStrings()` case)